app/assets/sass/application.scss

```diff
@@ -366,6 +366,12 @@ span.bold {
     float:right;
 }
 
+.pull-left{
```

Comment:
can you separate padding into another class so that we can reuse pull-left without it?